src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface TabsType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: TabsType[],
+  selectedTabsId: string,
+  onTabSelected: React.MouseEventHandler<HTMLAnchorElement>,
+  selectedTab: TabsType | undefined,
```

Comment:
You can remove the `selectedTab` prop because you already have `selectedTabsId`. You can find the selected tab content using `selectedTabsId` in the component.